docs/userGuide/deployingTheSite.md

```diff
@@ -21,15 +21,15 @@
 
 ## Generic steps for deploying a MarkBind site
 
-1. Set the [`baseUrl` property of the `site.json` file](siteJsonFile.html#baseUrl) to match the deploy location.
+1. Set the [`baseUrl` property of the `site.json` file](siteJsonFile.html#baseurl){.no-convert} to match the deploy location.
```

Comment:
Is there a reason why `no-convert` is needed here? I tried removing it and the link works fine without it.

Reply:
It shouldn't be there. Thanks for the catch!